CHANGELOG.md

```diff
@@ -5,13 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [4.3.0]
-### Added
-- An `iterative_min_size` parameter has been added to the `WATER_MAP_TEST` job spec.
 ### Changed
-- For `WATER_MAP_TEST` jobs, the default `minimization_metric` is now `ts`.
+- The `iterative_min_size` and `minimization_metric` parameters have been added to the `WATER_MAP` job spec, depreciating the `WATER_MAP_TEST` job spec. Notably, the default `minimization_metric` is now `ts`, instead of `fmi` as set in `WATER_MAP_TEST`
```

Comment:
```suggestion
- The `iterative_min_size` and `minimization_metric` parameters have been added to the `WATER_MAP` job spec, deprecating the `WATER_MAP_TEST` job spec. Notably, the default `minimization_metric` is now `ts`, instead of `fmi` as set in `WATER_MAP_TEST`
```

Reply:
I just *cannot* keep deprecate and depreciate straight :rofl: